frame/system/src/lib.rs

```diff
@@ -1272,6 +1272,40 @@ impl<T: Config> Pallet<T> {
 	/// Remove temporary "environment" entries in storage, compute the storage root and return the
 	/// resulting header for this block.
 	pub fn finalize() -> T::Header {
+		log::debug!(
+			target: "runtime::system",
+			"[{:?}] length: {} (normal {:?}, op: {:?}, mandatory {:?}) / normal weight: {} ({:?}) \
+			/ op weight {} ({:?}) / mandatory weight {} ({:?})",
+			Self::block_number(),
+			Self::all_extrinsics_len(),
+			sp_runtime::Percent::from_rational(
+				Self::all_extrinsics_len(),
+				*T::BlockLength::get().max.get(DispatchClass::Normal)
+			),
```

Comment:
What should this express?

Reply:
What portion of the block that's available to Normal extrinsics' length is full.

Reply:
I'm an idiot :P